client/src/services/sunfishapi.service.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+app.factory('SunfishSrvc', ['$http', function($http){
+    var baseUrl = '/api/siafile/';
+    var sunfishSrvc = {};
```

Comment:
in general I think it harms readability to shorten wordnames. A few things like src and dest are common enough that they practically belong in the dictionary, but stuff like srvc is confusing.